WordPress/Classes/ViewRelated/Reader/ReaderFollowedSitesViewController.swift

```diff
@@ -264,20 +264,20 @@ extension ReaderFollowedSitesViewController : WPTableViewHandlerDelegate
     }
 
 
-    func tableView(tableView: UITableView!, cellForRowAtIndexPath indexPath: NSIndexPath!) -> UITableViewCell! {
+    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCellWithIdentifier(cellIdentifier) ?? WPTableViewCell(style: .Subtitle, reuseIdentifier: cellIdentifier)
 
         configureCell(cell, atIndexPath: indexPath)
         return cell
     }
 
 
-    func tableView(tableView: UITableView!, heightForRowAtIndexPath indexPath: NSIndexPath!) -> CGFloat {
+    func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
         return 54.0
     }
 
 
-    func titleForHeaderInSection(section: Int) -> String! {
+    func titleForHeaderInSection(section: Int) -> String {
```

Comment:
If we make the protocol method an optional, it would be nice to leave this one explicitly unwrapped since it never returns `nil`.